src/agent/main.cpp

```diff
@@ -119,7 +121,10 @@ static otbrLogLevel GetDefaultLogLevel(void)
 
 static void PrintRadioVersion(otInstance *aInstance)
 {
-    printf("%s\n", otPlatRadioGetVersionString(aInstance));
+    const char *radioVersion = otPlatRadioGetVersionString(aInstance);
+
+    otbrLogNotice("Radio version: %s", radioVersion);
+    printf("%s\n", radioVersion);
```

Comment:
move this block into PrintRadioVersionAndExit()? I don't see this function is used anywhere else.